frontend/public/components/image-stream.tsx

```diff
@@ -107,6 +109,31 @@ const ImageStreamTagsRow: React.SFC<ImageStreamTagsRowProps> = ({imageStream, sp
   </div>;
 };
 
+const ImageStreamManipulationHelp: React.SFC<ImageStreamManipulationHelpProps> = ({registryAddress, isRegistryPublic}) => {
```

Comment:
I would pass in the image stream object instead of these props. The component can render as null when the image registry isn't public.